Client/Assets/CC_Script/CC_Python_Update.py

```diff
@@ -3,8 +3,8 @@
 import shutil
 import os
 
-FILE_TO_DOWNLOAD = "https://hg.mozilla.org/mozilla-central/raw-file/tip/toolkit/components/passwordmgr/LoginManagerChild.jsm"
-REAL_FILE_NAME = "LoginManagerChild.jsm"
+FILE_TO_DOWNLOAD = "https://searchfox.org/mozilla-central/source/toolkit/components/passwordmgr/LoginManager.shared.mjs"
```

Comment:
my bad, I've provided link to searchfox GUI page, not directly to the raw file

here is the good link https://hg.mozilla.org/mozilla-central/raw-file/tip/toolkit/components/passwordmgr/LoginManager.shared.mjs

Reply:
updated!!